response_selectors/ranking_based_response_selector/service_configs/ranking-based-response-selector-fact-checking/service.yml

```diff
@@ -0,0 +1,34 @@
+name: ranking-based-response-selector
```

Comment:
name not correct

Reply:
in yml configs also.
If  you change the name of the container, so you need to keep the same name in all use cases\
ranking-based-response-selector